src/main/java/medicaldoctor/util/S_Login.fxml.fxml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
what is that filename? `.fxml.fxml`? also try not to abbreviate, if this is secondary login, it should say so, not "S". Also I have been using the name "Additional" instead of secondary. It should be consistent. I can change my stuff to "Secondary".

Reply:
on the other hand you have "Additional" in the controller name below, maybe you should change your names.